Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java

```diff
@@ -0,0 +1,77 @@
+package com.dwg_karrier.roys;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+import java.util.ArrayList;
+
+public class DataBaseOpenHelper extends SQLiteOpenHelper {
+  public static final int DATABASE_VERSION = 1;
+  public static final String DATABASE_NAME = "FeedReader.db";
+
+  public DataBaseOpenHelper(Context context) {
+    super(context, DATABASE_NAME, null, DATABASE_VERSION);
+  }
+
+  public void onCreate(SQLiteDatabase db) {
+    db.execSQL(DataBases.CreateDB._CREATE);
+  }
+  public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
+    onCreate(db);
+  }
+
+  public int getTotalPageCount() {
+    SQLiteDatabase dataBase = getReadableDatabase();
+    int result = 0;
+
+    Cursor cursor = dataBase.rawQuery("SELECT count(*) from page ", null);
+    while (cursor.moveToNext()) {
+      result = cursor.getInt(0);
+    }
+
+    return result;
+  }
+
+  public int getReadedPageCount() {
```

Comment:
Past of read is read.
I think @sunju0123 's PR has the same code. Who's the owner?